src/content/api/promotions.mdx

```diff
@@ -0,0 +1,194 @@
+---
+title: Promotions
+description: Promotion model and related endpoints
+nav:
+  path: API/Loyalty Programs
+  order: 2
+---
+import Properties from '../../components/Properties.astro';
+import Property from '../../components/Property.astro';
+import Endpoint from '../../components/Endpoint.astro';
+import Badge from '../../components/Badge.astro';
+import Error from '../../components/Error.astro';
+
+Promotions are a mechanism to reward accounts for completing certain actions.
+
+## Promotion Model
+
+<Properties>
+  <Property name="id" type="string">
+    The unique identifier for the Promotion.
+  </Property>
+
+  <Property name="accountId" type="string">
+    The ID of the [Account](/api/accounts) that owns the Promotion.
+  </Property>
+
+  <Property name="name" type="string">
+    The display name of the Promotion.
+  </Property>
+
+  <Property name="programId" type="string">
+    The ID of the [Loyalty Program](/api/loyalty-programs) that the promotion belongs to.
+  </Property>
+
+  <Property name="description" type="string">
+    The display description for the Promotion.
+  </Property>
+
+  <Property name="mediaUploadId" type="string">
+    The ID of the [Media Upload](/api/media-uploads/) image of the Promotion.
+  </Property>
+
+  <Property name="startsAt" type="timestamp">
+    The timestamp of when the Promotion starts.
+  </Property>
+
+  <Property name="endsAt" type="timestamp">
+    The timestamp of when the Promotion ends.
+  </Property>
+
+  <Property name="rewards" type="array">
+    A list of [Rewards](/api/promotions#promotion-reward-model) that the account will receive upon completing the Promotion.
+  </Property>
+
+  <Property name="eventType" type="string">
+    The type of Event that will trigger increments and completions of the Promotion.
+  </Property>
+
+  <Property name="target" type="object">
+    Describes how to complete the promotion. See [Promotion Target](/api/promotions#promotion-target-model).
+  </Property>
+
+  <Property name="type" type="string">
+    The type of Promotion. Can be `cashback` or `challenge`. `challenge` gives a set reward one time. `cashback` gives a reward based on the payment amount, which can be completed an unlimited amount of times.
+  </Property>
```

Comment:
Suggestion: The type of Promotion. Can be cashback or challenge. A challenge rewards the account once, after the challenge conditions are met. A cashback gives a reward based on a payment amount, and can happen an unlimited number of times.

Reply:
I don't like the "after the challenge conditions are met" part. Both challenge and cashback type promotions can have conditions. I think I would change it to "after the challenge target is met".